Dockerfile

```diff
@@ -348,6 +362,8 @@ RUN if [ ! -z "${TIMESCALEDB_TOOLKIT_EXTENSION}" -a -z "${OSS_ONLY}" ]; then \
 
 USER root
 
+RUN du -ms /build/cache
+
```

Comment:
```suggestion
```

Reply:
Yeah, WIP, will clean up

Reply:
(Pushing some `wip` commits every so often as the purpose is to speed up CI/CD, this is to validate that assumption)